dts/arm/nxp/nxp_mcxw72.dtsi

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2024-2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nxp_mcxw7x_common.dtsi"
+
+&fmu {
+	ranges = <0x0 0x10000000 DT_SIZE_M(2)>;
+};
+
+&flash {
+	reg = <0x0 DT_SIZE_M(2)>;
+};
+
+&ctcm {
+	ranges = <0x0 0x14000000 DT_SIZE_K(32)>;
+};
+
+&ctcm0 {
+	reg = <0x0 DT_SIZE_K(32)>;
+};
+
+&stcm {
+	ranges = <0x0 0x30000000 DT_SIZE_K(232)>;
+};
+
+&stcm0 {
+	reg = <0x0 DT_SIZE_K(232)>;
+};
+
+&pbridge2 {
+	reg = <0x0 0x54000>;
+};
+
+&porta {
+	reg = <0x42000 0x20>;
```

Comment:
why is the size different, is there more registers on mcxw71 ? I am wondering how many of these `reg` are actually the same

Reply:
Updated the dts_common to reflect these sizes.